modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/display/context/RoleDisplayContext.java

```diff
@@ -489,12 +508,11 @@ private Map<String, String> _getTabsURLs() throws Exception {
 		).build();
 	}
 
-	private static final String[] _ASSIGNEE_TYPE_NAMES = {
-		"users", "sites", "organizations", "user-groups", "segments"
-	};
-
 	private Boolean _accountRoleGroupScope;
 	private Boolean _allowGroupScope;
+	private String[] _assigneeTypeNames = {
```

Comment:
Leave this variable as "constant" and use a local one to "store" the new array without _segment_ if administrator

Reply:
I needed to create a new method as the local variable was not being referenced when NavigationItemList was returned 
https://github.com/liferay-commerce/liferay-portal/pull/4499/commits/3e74aec33c9421e2e114a3fb50a3ce34a85c62b8#diff-0430dbaecaa63fc25fe61f0406791e480443aea0aad90681cfcd5aefc89a31b5R375